package.json

```diff
@@ -33,7 +33,7 @@
     "core-js": "^3.21.1",
     "fast-glob": "^3.2.7",
     "foundation-emails": "^2.3.1",
-    "identity-style-guide": "^6.7.0",
+    "@18f/identity-design-system": "file:../identity-style-guide",
```

Comment:
this is WIP/do not merge right?

Reply:
Yeah, this is just so I can test locally.

Reply:
Updated to published beta in 20333c7c71b3b092780ca194b3916fd8ea697091. Will change to stable release once it's published after testing.